overlord/hookstate/ctlcmd/model.go

```diff
@@ -178,7 +222,9 @@ func (c *modelCommand) Execute([]string) error {
 			return err
 		}
 	} else {
-		modelFormatter := modelCommandFormatter{}
+		modelFormatter := modelCommandFormatter{
+			snapInfo: snapInfo,
+		}
 		if err := clientutil.PrintModelAssertion(w, *deviceCtx.Model(), nil, modelFormatter, opts); err != nil {
```

Comment:
we need to pass in the serial assertion associated with the model if it exists, otherwise this will also print that the is no serial even if there is, we need to find the serial with something like:

```
  assertstate.DB(st).FindMany(asserts.SerialType, map[string]string{
                "brand-id":           <brand>,
                "model":              <model>,
  ...
```

this in theory could return many values, picking the newest is probabl good enough here (Serial has Timestamp).

Reply:
Ah nice, I have added functionality to find the serial assertion and provide that - I would love a review of how I do this.